### PR TITLE
Add missing section name to smart proxy sync test paragraph

### PR DESCRIPTION
**Pull request by jhutar**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/modules/con_smartproxy_performance_tests.adoc [hunk 1]
```diff
@@ -7,8 +7,8 @@ We have measured multiple test cases on multiple {SmartProxy} configurations:
 |===
 |{SmartProxy} HW configuration |CPUs |RAM
 |minimal |4 |12 GiB
-|large |8 |24 GiB
-|extra large |16 |46 GiB
+|large |8 |24 GiB
+|extra large |16 |46 GiB
```
Comment (maximiliankolb) [inline guides/common/modules/con_smartproxy_performance_tests.adoc:11]: Is this change removing tabs in favor of spaces? Github UI doesn't tell.

Comment (jhutar) [reply guides/common/modules/con_smartproxy_performance_tests.adoc:11]: Yep, exactly, that is it

Comment (jhutar) [reply guides/common/modules/con_smartproxy_performance_tests.adoc:11]: Or maybe it was not Tab, but some other non-space whitespace char

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: ✅ 
   76bc4771a..a34b170d4  3.4 -> 3.4
   3eeb4dd5d..62834be7c  3.3 -> 3.3
   c3d46ad09..f1042db68  3.2 -> 3.2
   1c1f9b49c..8a44df554  3.1 -> 3.1

**Pull request merged**